beanquery/shell.py

```diff
@@ -157,24 +158,23 @@ def get_pager(self):
         if self.is_interactive:
             return pager.ConditionalPager(self.vars.get('pager', None),
                                           minlines=misc_utils.get_screen_height())
-        else:
-            file = (codecs.getwriter("utf-8")(sys.stdout.buffer)
-                    if hasattr(sys.stdout, 'buffer') else
-                    sys.stdout)
-            return pager.flush_only(file)
+        file = (codecs.getwriter("utf-8")(sys.stdout.buffer)
+                if hasattr(sys.stdout, 'buffer') else
+                sys.stdout)
+        return pager.flush_only(file)
 
-    def cmdloop(self):
+    def cmdloop(self, intro=None):
         """Override cmdloop to handle keyboard interrupts."""
         while True:
             try:
-                super().cmdloop()
+                super().cmdloop(intro)
                 break
             except KeyboardInterrupt:
                 print('\n(Interrupted)', file=self.outfile)
 
-    def do_help(self, command):
+    def do_help(self, arg):
```

Comment:
W0221: Parameters differ from overridden 'cmdloop' method (arguments-differ)

Reply:
Here too. I would disable the warning in the pylint configuration. Please also see my comment about using beancount pylint configuration instead than the beangulp one.

Reply:
I disable a bunch of checks in Beancount mainly because I haven't spent the time to fix them all.
I tend to agree with most of the pylint recommendations (though a few of them are annoying); I'd like incrementally trim down the disabling of the checks to reduce the configuration slowly toward the default.